### PR TITLE
[Keymap] Lily58 : HELL0 NAVI. Interface

### PR DESCRIPTION
**Pull request by druotoni**
## Description

Scifi interface for lily58 with animations on both sides.
It's my first real commit, so I hope it's ok.
Thanks !

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lily58/keymaps/druotoni/rules.mk [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight. 
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+OLED_DRIVER_ENABLE= yes     # OLED display
```
Comment (drashna) [inline keyboards/lily58/keymaps/druotoni/rules.mk:18]: All of the rest of these match the defaults, so don't need to be set. 
And `OLED_DRIVER_ENABLE` is depreciated anyways. 

```suggestion
RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight. 
```

File: keyboards/lily58/keymaps/druotoni/rules.mk [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+# EXTRAFLAGS += -flto
+LTO_ENABLE = yes
+SPACE_CADET_ENABLE = no
+GRAVE_ESC_ENABLE = no
+MAGIC_ENABlE  = no
```
Comment (drashna) [inline keyboards/lily58/keymaps/druotoni/rules.mk:29]: whoops: 
```suggestion
MAGIC_ENABLE  = no
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/lily58/keymaps/druotoni/burst.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// Copyright 2020 Richard Sutherland (rich@brickbots.com)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/burst.h:4]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/boot.h [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/boot.h:3]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/draw_helper.h [hunk 5]
```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// Copyright 2021 ugfx
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/draw_helper.h:4]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/fast_random.h [hunk 6]
```diff
@@ -0,0 +1,5 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/fast_random.h:3]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/gui_state.h [hunk 7]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/gui_state.h:3]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/layer_frame.h [hunk 8]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/layer_frame.h:3]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/navi_logo.h [hunk 9]
```diff
@@ -0,0 +1,5 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/navi_logo.h:3]:
```suggestion

#pragma once

```

File: keyboards/lily58/keymaps/druotoni/ring.h [hunk 10]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 Nicolas Druoton (druotoni)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/ring.h:3]:
```suggestion

#pragma once

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/lily58/keymaps/druotoni/keymap_french.h [hunk 11]
```diff
@@ -0,0 +1,165 @@
+/* Copyright 2015-2016 Jack Humbert
```
Comment (zvecr) [inline keyboards/lily58/keymaps/druotoni/keymap_french.h:1]: This file should be deleted, as its just <https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_french.h>

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]
`__attribute__ ((weak))` :heavy_check_mark:

**Pull request merged**